packages/components/src/styles/paragraph/paragraph.stories.ts

```diff
@@ -1,20 +1,18 @@
 import '../../solid-components';
-
+import { html } from 'lit-html';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
 
 const { argTypes, parameters } = storybookDefaults('sd-paragraph');
 const { overrideArgs } = storybookHelpers('sd-paragraph');
 const { generateTemplate } = storybookTemplate('sd-paragraph');
 
 /**
- * A paragraph is used to display blocks of text. It uses the base font size and can contain bold and/or link styles.<br>
- * <br>
- * <b>Sizes</b>
- * <li>lg is the default paragraph size.</li>
- * <li>sm can be used as an alternative for tighter spaces.</li>
+ * Used to display blocks of text.
+ * It uses the base font size and can contain bold and/or link styles.
  */
```

Comment:
For the 1st sample can you make it look like FIGMA?

"Large Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique. Facilisis commodo integer hendrerit tortor."

Reply:
Done